prime-router/metadata/fhir_mapping/hl7/datatype/ContactPoint.yml

```diff
@@ -80,6 +80,13 @@ use_4:
   vars:
     useCode: String, XTN.2
 
+use_5:
+  condition: $useCode EQUALS ORN
+  type: STRING
+  value: "temp"
+  vars:
+    useCode: String, XTN.2
+
```

Comment:
Where does this mapping come from? Do you know why we have a test for "temp"?

Reply:
I got the mapping from how the test was set up and the FHIR -> HL7 mapping found below.

https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/metadata/hl7_mapping/common/datatype/xtn-telecom.yml#L14

Reply:
The particular mapping for ORN was introduced in [this commit](https://github.com/CDCgov/prime-reportstream/commit/fa3db4c1e1025d583df06d317fb2037ab1e44dff#diff-7cf6b3e50bf178ab7427e314589d85b4c3747ea2e8231c654385fa5afa3af784), but it's not clear why. It's not spelled out in [this sheet](https://docs.google.com/spreadsheets/d/1EE3a5UIGXLi8wMDr49fWTEVx8QS2GFqYgitE3vLOQTQ/edit#gid=0), but `other` seems more appropriate. Maybe `temp` was meant to be a temporary value?

Reply:
Still don't understand why temp exists in the mapping in the first place. But that's outside scope here so makes sense add the element.